cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -122,9 +122,16 @@ function up() {
     eval ${_cli:?} run $params
 
     # Copy k8s config and kubectl
-    ${_cli} scp --prefix ${provider_prefix:?} /usr/bin/kubectl - >${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
+    # Workaround https://github.com/containers/conmon/issues/315 by not dumping the file to stdout for the time being
+    if [ ${_cri_bin} == "podman" ]; then
+        ${_cli} scp --prefix ${provider_prefix:?} /usr/bin/kubectl /kubevirtci_config/.kubectl
```

Comment:
What I do not fully understand: `gocli ssh` is implementing the `scp` protocol and directly connecting to the SSH server inside the VM. it is not entirely clear to me how stdout from another process can interfere with this, even after reading your issue.

Reply:
Ah never mind. checked the `scp` implementation again. Makes sense.